alignak/dependencynode.py

```diff
@@ -84,7 +85,7 @@ def __init__(self, params=None):
             if 'operand' in params:
                 self.operand = params['operand']
             if 'sons' in params:
-                self.sons = [DependencyNode(elem) for elem in params['sons']]
+                self.sons = [unserialize(elem) for elem in params['sons']]
```

Comment:
Forget this ... we cannot even get a valid configuration :/ unserializing is necessary!